cdc/processor/processor.go

```diff
@@ -185,14 +185,6 @@ func (p *processor) AddTable(
 			zap.Bool("isPrepare", isPrepare))
 	}
 
-	log.Info("adding table",
```

Comment:
this information is important ,we shouldn't ignore  it. Perhaps we can skip it in too many tables case.

Reply:
we can know `adding table` this event by other logs.